src/kemal/route_handler.cr

```diff
@@ -24,21 +24,14 @@ class Kemal::RouteHandler < HTTP::Handler
 
   # Processes the route if it's a match. Otherwise renders 404.
   def process_request(context)
-    url = context.request.path.not_nil!
+    raise NilPathException.new unless context.request.path
```

Comment:
I read the crystal doc api and found that `.not_nil!` raises an exception I just thought a clearer exception would be better.